### PR TITLE
settings/core: Document ssl_cipher_suites

### PR DESCRIPTION
**Pull request by cmouse**
JIRA: DOV-4306

### CODE REVIEW

File: source/settings/core.rst [hunk 1]
```diff
@@ -4291,6 +4291,16 @@ See :ref:`dovecot_ssl_configuration`
...
+.. _setting-ssl_cipher_suites:
+
+``ssl_cipher_suites``
+---------------------
+
+ - Default: OpenSSL version specific.
+
+.. versionadded:: v2.3.15
+
+The list of SSL cipher suites to use, in order of preference.
```
Comment (sirainen) [inline source/settings/core.rst:4303]: Could have some kind of example values. and/or link to OpenSSL documentation about it.

File: source/settings/core.rst [hunk 2]
```diff
@@ -4291,6 +4291,16 @@ See :ref:`dovecot_ssl_configuration`
...
 .. todo:: Indicate SSL setting
 
+.. _setting-ssl_cipher_suites:
+
+``ssl_cipher_suites``
+---------------------
+
+ - Default: OpenSSL version specific.
+
+.. versionadded:: v2.3.15
```
Comment (sirainen) [inline source/settings/core.rst:4301]: versionadded should be before "default" like it is elsewhere

**Review by sirainen — CHANGES_REQUESTED** [review]

**Comment by sirainen** [comment]
/github/workspace/source/settings/core.rst:4305: WARNING: Unknown target name: "https://wiki.openssl.org/index.php/tls1.3#ciphersuites".

**Comment by cmouse** [comment]
need to check syntax...

**Review by sirainen — APPROVED** [review]

**Pull request merged**